### ISSUE TITLE
Suggestion: Maybe a close grain button on the top bar?

### ISSUE THREAD

**Issue by ocdtrekkie** [sandstorm-io/sandstorm#1254]
Some early experience with the new sidebar makes me think... having to go to the expand arrow just to close a grain (and then re-contract the sidebar afterwards) seems like moving a one-step process to a three-step process.

When I want to close something, my cursor does instinctually go up. Would it be warranted to put a close grain button up on the top bar near the other grain actions?

**Comment by kentonv** [comment]
FWIW, I like this idea, but @zarvox doesn't like having multiple buttons that do the same thing.

**Comment by ocdtrekkie** [comment]
Four years experience later: I tend to keep the sidebar expanded, mostly so I can easily close grains. Maybe it's just because I am not needing the horizontal space that badly in retrospect.

If one was to implement this someday, perhaps it would be correct to only show the close grain button on the top bar when the sidebar was minimized?

**Comment by zenhack** [comment]
I agree on desktop I haven't felt much need for more horizontal screen space. This was even true on my venerable 2009 thinkpad x200 before it died (in 2018?).

On mobile, where real estate is actually at a premium, you have to expand the menu to get at the stuff that would be in the top bar anyway.

+1 for keeping things simple and not adding redundancy to the UI.
